examples/liveness/README.md

```diff
@@ -0,0 +1,75 @@
+
+The **http-liveness.yaml** demonstrates the HTTP check.
+```
+    livenessProbe:
+      httpGet:
+        path: /healthz
+        port: 8080
+      initialDelaySeconds: 15
+      timeoutSeconds: 1
+```
+The Kubelet sends a HTTP request to the speicified path and port to perform the health check. If you take a look at image/server.go, you will see the server starts to respond with an error code 500 after 10 seconds, so the check fails.
+
+This [guide](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/examples/walkthrough/k8s201.md#health-checking) has more information on health checks.
+
+## Get your hands dirty
+To show the health check is actually working, first create the pods:
+```
+# cluster/kubectl.sh create -f exec-liveness.yaml
+# cluster/kbuectl.sh create -f http-liveness.yaml
+```
+
+Check the status of the pods once they are created:
+```
+# cluster/kubectl.sh get pods
+POD             IP           CONTAINER(S)   IMAGE(S)                            HOST                                     LABELS          STATUS    CREATED     MESSAGE
+liveness-exec   10.244.3.7                                                      kubernetes-minion-f08h/130.211.122.180   test=liveness   Running   3 seconds   
+                             liveness       gcr.io/google_containers/busybox                                                             Running   2 seconds   
+liveness-http   10.244.0.8                                                      kubernetes-minion-0bks/104.197.10.10     test=liveness   Running   3 seconds   
+                             liveness       gcr.io/google_containers/liveness                                                            Running   2 seconds   
+```
+
+Check the status half a minute later, you will see the termination messages:
+```
+# cluster/kubectl.sh get pods
+POD             IP           CONTAINER(S)   IMAGE(S)                            HOST                                     LABELS          STATUS    CREATED      MESSAGE
+liveness-exec   10.244.3.7                                                      kubernetes-minion-f08h/130.211.122.180   test=liveness   Running   34 seconds   
+                             liveness       gcr.io/google_containers/busybox                                                             Running   3 seconds    last termination: exit code 137
+liveness-http   10.244.0.8                                                      kubernetes-minion-0bks/104.197.10.10     test=liveness   Running   34 seconds   
+                             liveness       gcr.io/google_containers/liveness                                                            Running   13 seconds   last termination: exit code 2
+```
+The termination messages indicates that the liveness probe failed, and the container has been killed and recreated.
+
+You can see that the container is killed and restarted using *kubectl describe*,
```

Comment:
You can also see the container restart count being incremented by running `kubectl describe`.